src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+.stars__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  background-image: url(images/star.svg);
```

Comment:
The image path should be '../images/star.svg' assuming that your CSS file is in a 'src' folder and your images are in 'src/images'. This is necessary to correctly locate the image file relative to the CSS file.